cmd/prepare-storage.go

```diff
@@ -137,7 +137,7 @@ func prepForInitXL(firstDisk bool, sErrs []error, diskCount int) InitActions {
 		return WaitForConfig
 	}
 
-	quorum := diskCount/2 + 1
+	quorum := diskCount / 2
```

Comment:
``quorum`` should continue to represent the number of 'votes' required for a simple majority. For conditions where we require values different from quorum but relative to it, we can express them as e.g ``quorum-1``. This is to avoid different places in the code holding different initial values for variables representing the concept of quorum.